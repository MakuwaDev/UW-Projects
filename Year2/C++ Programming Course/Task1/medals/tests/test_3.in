=1 1 1
U K 3
U S A 2
Germany 1

Rzeczpospolita Polska 1
Rzeczpospolita Polska 1
Rzeczpospolita  Polska 2
Rzeczpospolita  Polska 2
A cokolwiek 0
=100 10 1