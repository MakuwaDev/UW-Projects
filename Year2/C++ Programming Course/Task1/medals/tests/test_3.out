1. Rzeczpospolita Polska
2. Germany
3. Rzeczpospolita  Polska
4. U S A
5. U K
6. A cokolwiek
